### ISSUE TITLE
03_linear_regression_sol.py for quadratic case

### ISSUE THREAD

**Issue by jluo80** [chiphuyen/stanford-tensorflow-tutorials#54]
I tried to add an additional variable 'u' as the notes_03.pdf suggested, however the loss and parameters became very large so that the final output is 'nan'. I tried to use another learning rate, e.g 1e^10 which can help to get specific values. However, I don't know why the loss will be very large (inf) with the square loss function. I mean even if the learning rate is very big, the loss will fluctuate between a range instead of getting out of limit.

**Comment by MianB1zhe** [comment]
I meet the same question 'nan' as you in the first, but then i modify my code (now i forget where i change) and it runs well. just now i change the learning rate to 1e1 and 1e10, and the loss did fluctuate between a very large range and has a  decrease trending. so i think maybe somewhere has an error in your code.

**Comment by jluo80** [comment]
I figured it out and it has no problem with my code. The learning rate is the main reason that cause this problem. After I reviewed the notes provided by Andrew Ng, the classic machine learning course. I found that the loss value can jump out of the bottom of the quadratic curve.
